src/services/import-service.js

```diff
@@ -59,8 +59,12 @@ class ImportService {
       .map(exportableDocument => {
         const importedDocument = importedDocuments.find(document => document.key === exportableDocument.key);
 
+        let importType = DOCUMENT_IMPORT_TYPE.add;
+        if (importedDocument) {
+          importType = DOCUMENT_IMPORT_TYPE.update;
+        }
         if (importedDocument?.revision === exportableDocument.revision) {
```

Comment:
I would move this to line 64 and make it a ternary between an update and reimport.

Reply:
Good idea, done and also now moved setting `importType = DOCUMENT_IMPORT_TYPE.add` on the `else` branch cause it was bothering me that I had to pre-set the variable to it.